.github/workflows/chez-build.yml

```diff
@@ -53,6 +52,7 @@ jobs:
         run: |
           ./configure --pb
           make -j$(($(nproc) + 1)) -l$(nproc) ${{ matrix.mach }}.bootquick
+      # ???
```

Comment:
@pmatos 

I'm a bit confused here. Should we also run "Test Chez" here too?
It looks like we are discarding the build from PB boot files.
(It's not a total effort waste because compilation error would still show up, but the compilation result seems to be unused otherwise).

Reply:
This is correct already, as detailed in https://github.com/racket/ChezScheme/blob/master/BUILDING